packages/grafana-ui/src/components/Forms/Label.tsx

```diff
@@ -3,7 +3,7 @@ import { useTheme, stylesFactory } from '../../themes';
 import { GrafanaTheme } from '@grafana/data';
 import { css, cx } from 'emotion';
 
-export interface LabelProps extends React.HTMLAttributes<HTMLLabelElement> {
+export interface LabelProps extends React.LabelHTMLAttributes<HTMLLabelElement> {
```

Comment:
What's the difference between these two?

Reply:
`LabelHTMLAttributes` has `htmlFor` prop on it